### DISCUSSION TITLE
Can we implement  openid-client with angular 11 ?

### DISCUSSION THREAD

**Discussion by abdelamine06** [panva/openid-client#376] [Q&A]
I try to implement SSO authentification with openid-client and angular, but when I create my service and try to inject it in my login component, I have a issue with webpack, so I do not know if a problem with angular or openid-client ? thanks for help

**Comment by panva** [comment]
> openid-client is a server side OpenID Relying Party (RP, Client) implementation for Node.js runtime

No you will not be able to bundle it for use in a browser.

**Reply by abdelamine06** [reply]
Okey, thank you
